### PR TITLE
RIA-691 Manual address where postcode not known

### PR DESCRIPTION
**Pull request by fantasticjamieburns**

### CODE REVIEW

File: e2e/features/address-manual.feature [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    And I complete the Home Office reference page
+    And I complete the Basic details page
+
+  @create-case @lodge-appeal@alternate @RIA-691
+  Scenario: Lodge an appeal application with a manually entered address
+
+    Given I am signed in as a Legal Rep
+    Given I create a new case
+    Given I complete the Home Office reference page
+    Given I complete the Basic details page
```
Comment (Rob-Doherty) [inline e2e/features/address-manual.feature]: These 4 steps have already been defined in the `Background:` section, so should be removed

File: e2e/features/address-manual.feature [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    And I click the Continue button
+
+    Given I complete the Why is your client appealing? page
+    Given I complete the New matters page
+    Given I complete the Has your client appealed against any other UK immigration decisions? page
+    Given I complete the Your own reference number page
+    When I click the Save and continue button
+    Then I should see an alert confirming the case has been created
+
+    Given I click the Case details tab
```
Comment (Rob-Doherty) [inline e2e/features/address-manual.feature]: This would probably make more sense as a `When`

File: e2e/features/address-manual.feature (hunk 1 above)
Comment (fantasticjamieburns) [reply e2e/features/address-manual.feature]: 👍

File: e2e/features/address-manual.feature (hunk 2 above)
Comment (fantasticjamieburns) [reply e2e/features/address-manual.feature]: 👍

**Pull request merged**